Microsoft.Toolkit.Sample.Forms.App/Microsoft.Toolkit.Sample.Forms.App.csproj

```diff
@@ -34,8 +34,12 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.0'">
-    <PackageReference Include="System.Runtime.WindowsRuntime" Version="4.6.0-preview.19073.11" />
-    <PackageReference Include="System.Runtime.WindowsRuntime.UI.Xaml" Version="4.6.0-preview.19073.11" />
+    <Reference Include="System.Runtime.WindowsRuntime">
```

Comment:
Why is this change necessary?

Reply:
I wont build in my local dev box, and it also make it consistent with other projects.

---
In reply to: [261019010](https://github.com/windows-toolkit/Microsoft.Toolkit.Win32/pull/85#discussion_r261019010) [](ancestors = 261019010)

Reply:
This is probably going away when we move to the new NuGet based SDK, so I'm ok with it.